lib/mayaUsd/utils/utilSerialization.h

```diff
@@ -64,6 +64,33 @@ enum USDUnsavedEditsOption
 MAYAUSD_CORE_PUBLIC
 USDUnsavedEditsOption serializeUsdEditsLocationOption();
 
+/*! \brief Return if the relative-path plug is set to true on the proxy shape.
+ */
+MAYAUSD_CORE_PUBLIC
+bool isProxyShapePathRelative(MayaUsdProxyShapeBase& proxyShape);
+
+/*! \brief File path mode for the setNewProxyPath and isProxyPathModeRelative functions.
+ *         kProxyPathRelative makes the file path relative to the Maya scene.
+ *         kProxyPathAbsolute makes the file path absolute.
+ *         kProxyPathFollowProxyShape reads the relative-path plug of the proxy shape
+ *                                    to decide.
+ *         kkProxyPathFollowOptionVar reads the mayaUsd_MakePathRelativeToSceneFile
+ *                                    options to decide.
+ */
+enum ProxyPathMode
+{
+    kProxyPathRelative,
+    kProxyPathAbsolute,
+    kProxyPathFollowProxyShape,
+    kkProxyPathFollowOptionVar
+};
+
+/*! \brief Convert the proxy path mode into a boolean telling if the path should relative.
```

Comment:
minor typo close to the end.

Reply:
I'll take note of it for a future PR, seems minor to respin everything.

Reply:
kkProxyPathFollowOptionVar also has two "k"